src/libs/image.c

```diff
@@ -583,7 +583,7 @@ void gui_init(dt_lib_module_t *self)
 
   dt_lib_module_t *meta = (dt_lib_module_t *) dt_action_section(DT_ACTION(self), N_("metadata"));
   d->copy_metadata_button = dt_action_button_new(meta, N_("copy"), copy_metadata_callback, self,
-                                                 _("set the (first) selected image as source of metadata"), 0, 0);
+                                                 _("set the selected image as source of metadata"), 0, 0);
```

Comment:
not sure it is more accurate, we are here really speaking of the first image selected in the lighttable where we can have multiple selected.

Reply:
In the lighttable we can have multiple images selected, indeed. But once we have selected multiple images this action (metada copying) will become unavailable. So it's just not true that the first selected image (out of several) can be a source of metadata. The source of metadata can only be a single (single, not the first!) selected image and it should be worded that way. The current wording is simply misleading.

Reply:
Ah, indeed! Now that we have a responsive interface it is indeed not possible to copy while multiple images are selected. All good then, thanks.